File: Python/Deficient_Number.py
print("Deficient Number Checker\n")
n = int(input("Enter the number : "))
s = 0
for i in range(1, n//2+1) :
	if n%i == 0:
		s += i
if s < n:
	print(f"{n} is a deficient number.")
else:
	print(f"{n} is not a deficient number.")

File: Java/src/Deficient_Number.java
/**
 * Deficient Number Checker
 * 
 * @author Jaysmito Mukherjee 
 * @version 2019-A-011
 */
import java.util.*;
public class Deficient_Number
{
    public static void main()
    {
        System.out.print("\f");
        Scanner sc=new Scanner(System.in);
        System.out.print("Deficient Number Checker\n~~~~~~~~~ ~~~~~~ ~~~~~~~\nEnter the number: ");
        int n=sc.nextInt();
        int s=0;
        for(int i=1;i<=n/2;i++)
            if(n%i==0)
            s=s+i;
        if(s<n)
            System.out.print(n+" is a Deficient number.");
        else
            System.out.print(n+" is not a Deficient number.");        
    }
}
